src/_guides/language/effective-dart/usage.md

```diff
@@ -1053,6 +1098,22 @@ actually return a new object.
 The language still permits `new` in order to make migration less painful, but
 consider it deprecated and remove it from your code.
 
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/usage_good.dart (no-new)" replace="/new //g"?>
+{% prettify dart %}
+Widget build(BuildContext context) {
+  return Row(
+    children: [
+      RaisedButton(
+        onPressed: increment,
+        child: Text('Increment'),
+      ),
+      Text('Count: $counter'),
+    ],
+  );
+}
+{% endprettify %}
+
```

Comment:
AngularDart doesn't yet support using `new`. Maybe add a note about this, with a link to https://github.com/dart-lang/angular/issues/1313?

Reply:
I revised the guideline to make it clear that it only applies if you don't need to support tools that don't implement it. I'm hesitant to mention Angular specifically because I don't want the doc to get out of date.